Code/Framework/AzCore/AzCore/Debug/Profiler.cpp

```diff
@@ -7,4 +7,67 @@
  */
 
 #include <AzCore/Debug/Profiler.h>
+#include <AzCore/Debug/ProfilerBus.h>
+#include <AzCore/Console/IConsole.h>
+#include <AzCore/Console/ILogger.h>
+#include <AzCore/Settings/SettingsRegistry.h>
 
+namespace AZ::Debug
+{
+    AZStd::string GenerateOutputFile(const char* nameHint)
+    {
+        AZStd::string timeString;
+        AZStd::to_string(timeString, AZStd::GetTimeNowSecond());
```

Comment:
`AZStd::GetTimeNowSecond()` returns a `AZStd::sys_time_t` which is just a `AZ::s64` so this intermediate string isn't needed and can be directly set in the format below. Alternatively a fixed sized string can be used to at least avoid the temporary string allocations as I'm not sure that the size of the entire time is guaranteed to be within the small string optimization.

Reply:
Good point, will fix.